src/utilities.lisp

```diff
@@ -79,9 +79,9 @@ WARNING: This routine is based on MAPHASH, which has undefined behavior if the s
 
 (defmacro ignorant-lambda (&body body)
   "Defines an anonymous function that discards all of its arguments."
-  (a:with-gensyms (time)
-    `(lambda (,time)
-       (declare (ignore ,time))
+  (a:with-gensyms (rest)
+    `(lambda (&rest ,rest)
```

Comment:
why this change?

Reply:
I just noticed that this macro isn't doing what it claimed: it doesn't discard its arguments wholesale, it asserts that they take a particular form and then discards them.